ethereum/swap_creator_test.go

```diff
@@ -441,12 +446,19 @@ func testRefundAfterT1(t *testing.T, asset ethcommon.Address, erc20Contract *Tes
 		Nonce:        nonce,
 	}
 
-	// now let's try to refund
 	var s [32]byte
 	secret := proof.Secret()
 	copy(s[:], secret[:])
 	tx, err = contract.Refund(auth, swap, s)
 	require.NoError(t, err)
+	_, err = block.WaitForReceipt(context.Background(), conn, tx.Hash())
+	require.ErrorContains(t, err, "VM Exception while processing transaction: revert")
+
+	<-time.After(time.Until(time.Unix(t1.Int64()+1, 0)))
```

Comment:
Did the +1 make a difference? More curious than anything else.

Reply:
yeah, it sometimes fails without it due to submitting a bit too early